core/db/migrate/20170412103617_transform_tax_rate_category_relation.rb

```diff
@@ -19,23 +31,26 @@ def up
   end
 
   def down
-    add_column :spree_tax_rates, :tax_category_id, :integer
+    add_column :spree_tax_rates, :tax_category_id, :integer, index: true
+    add_foreign_key :spree_tax_rates, :spree_tax_categories, column: :tax_category_id
+
+    TaxRate.find_each do |tax_rate|
+      tax_categories = TaxRateTaxCategory.where(tax_rate_id: tax_rate.id)
```

Comment:
Should this be:
```
tax_category_ids = TaxRateTaxCategory.where(tax_rate_id: tax_rate.id).pluck(:tax_category_id)
```
? The id is all you need and below you want the `spree_tax_categories.id` rather than the `spree_tax_rate_tax_categories.id` right?